pkg/volume/storageos/storageos_util.go

```diff
@@ -323,27 +346,30 @@ func getLoopDevice(path string, exec mount.Exec) (string, error) {
 	}
 
 	args := []string{"-j", path}
-	out, err := exec.Run(losetupPath, args...)
+	cmd := exec.Command(losetupPath, args...)
```

Comment:
Please add a comment that you intentionally run where kubelet is (e.g. in kubelet's container).

Reply:
@jsafrane thanks for the review, but quick question as I noticed that I reverted the VolumeHost.GetExec() change you made without fully understanding it.

While we expect the mount utilities (incl `losetup`) to run where the kubelet runs, we don't require it.  We just need the device and mount to be available in the correct place and where the kubelet can see them.

Unless you can think of any gotchas or I'm misunderstanding, I'll revert this bit.

Also fyi: the Block Volume support loop device code seems to have used this as a base, and doesn't use VolumeHost.GetExec().  Not sure if this was intentional, and if it was do I need the same here? https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/util_linux.go.  Maybe @mtanino can advise?

Reply:
In 1.9 there is a possibility to run mount utilities (mount.nfs, iscsiadm, mkfs.btrfs, ..., losetup) in a dedicated container instead of the host (which can be a minimal one like CoreOS) or in the container where kubelet runs (which can have just kubelet). The setup is quite elaborate and it is dedicated only for testing in our e2e tests, which run quite minimal GCI image (but it has losetup).   `VolumeHost.GetExec()` is used to execute things in these containers with mount utilities instead of on the host. You probably should use this interface for losetup too, just in case someone has really minimal OS without it.
 
In addition, it would be great if you could prepare container(s) with the server parts of StorageOS and e2e tests that would deploy it and test the plugin against. This way, we could test our plugin and make sure there are no regressions as the plugin evolves.

Reply:
Thanks, I'll revert this bit.  We're working on the e2e tests separately.  I'm hoping we can get a PR  for them in the next few weeks.

Reply:
I still see you use `exec.Command` directly, did you plan to revert it back to `mount.Exec` interface?